compute_endpoint/tests/unit/test_engines.py

```diff
@@ -73,6 +73,36 @@ def test_engine_submit(engine_type: GlobusComputeEngineBase, engine_runner):
     assert future.result(timeout=5) == param * 2
 
 
+@pytest.mark.parametrize(
+    "engine_type", (ProcessPoolEngine, ThreadPoolEngine, GlobusComputeEngine)
+)
+def test_engine_working_dir(
+    engine_type: GlobusComputeEngineBase,
+    engine_runner,
```

Comment:
From a test clarity standpoint, I observe that this test relies on an external detail that is not verified.  Namely, the `working_dir` argument.  Should we verify both a relative and absolute path as well?  Consider `@pytest.mark.parametrize`, with `"/some/abs/path"`, `"relative_dir/"` and `None`.